roles/confluent.common/tasks/failure_handling.yml

```diff
@@ -3,6 +3,7 @@
   file:
     state: directory
     path: "error_files/{{inventory_hostname}}"
+    mode: 0750
```

Comment:
Wondering about this one, this is a directory on the controller for error log files. maybe 755? wdyt?
When you git clone our repo, itll be 755

Reply:
The component log-directories are set to 750, and that has annoyed me personally. I struggeled a lot to allow splunk to harvest the logs. I guess it depends on the controller environment? We are running container controllers, so I do not have a strong opinion here.

Reply:
The solution there is to put these modes into variables... more variables though.
Lets go with `0755` here to keep it in line with permissions applied during git clone

Reply:
Agreed, done.